.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,30 @@
+name: Deploy Docs
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-deploy:
+    name: Build and deploy docs
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3
+        uses: actions/setup-python@v4
+
+      - name: Install dependencies 🔧
+        working-directory: docs
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install -r requirements.txt
+
+      - name: Build and deploy to GitHub Pages 🏗️ 🚀
+        working-directory: docs
+        run: |
+          mkdocs gh-deploy --force --strict
```

Comment:
makes sure there's no warnings when building the docs, or the workflow will fail

Reply:
e.g. linking documents which don't exist (many of those previously)